src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -95,11 +101,15 @@ static void DeleteOtherPlatformHost(Permutation permutation)
                 x86.Delete();
             }
         }
-
-        [TestFixtureSetUp]
-        public void RunBeforeAnyTests()
+    }
+    
+    [SetUpFixture]
+    public class TestSessionInitializer
+    {
+        [SetUp]
+        public void Setup()
         {
-            SessionId = Guid.NewGuid().ToString();
+            Base.SessionId = DateTime.UtcNow.Ticks.ToString();
```

Comment:
Also, trying to understand this:

>Attribute used to mark a class that contains one-time SetUp and/or TearDown methods that apply to all the tests in a namespace or an assembly.

Does this mean a new session ID per fixture, or for *all* fixtures?